mesheryctl/Makefile

```diff
@@ -2,40 +2,33 @@ make:
 	go build -o mesheryctl cmd/mesheryctl/main.go
 forwin:
 	go build -o mesheryctl.exe cmd/mesheryctl/main.go
-OUTDIR := bin
 
-ARCH := amd64
+# ---------------------------------------------------------------------------------
+# build for mutiple platform
 
-BINNAME_DARWIN ?= mesheryctl-darwin-$(ARCH)
-BINNAME_LINUX ?= mesheryctl-linux-$(ARCH)
-BINNAME_WINDOWS ?= mesheryctl-windows-$(ARCH).exe
+TAGS :=
+TARGETS :=darwin/amd64 linux/amd64 windows/amd64
+BINNAME ?=mesheryctl
+
+GOPATH = $(shell go env GOPATH)
+GOX = $(GOPATH)/bin/gox
 
 # go options
 
 # Need to remove the DWARF debug information and the symbol information
 LDFLAGS := -w -s
-
 # Load the parameters to go compiler and print optimization decisions
 GCFLAGS := 
 
+$(GOX):
+	(cd /; GO111MODULE=on go get -u github.com/mitchellh/gox)
 
-# CLI
-
-.PHONY: $(BINNAME_DARWIN)
-$(BINNAME_DARWIN):
-	@echo
-	CGO_ENABLED=0 GOARCH=$(ARCH) GOOS=darwin GO111MODULE=on go build -ldflags='$(LDFLAGS)' -gcflags='$(GCFLAGS)' -o $(OUTDIR)/$(BINNAME_DARWIN) ./cmd/mesheryctl/main.go
-
-.PHONY: $(BINNAME_LINUX)
-$(BINNAME_LINUX):
-	@echo
-	CGO_ENABLED=0 GOARCH=$(ARCH) GOOS=linux GO111MODULE=on go build -ldflags='$(LDFLAGS)' -gcflags='$(GCFLAGS)' -o $(OUTDIR)/$(BINNAME_LINUX) ./cmd/mesheryctl/main.go
 
-.PHONY: $(BINNAME_WINDOWS)
-$(BINNAME_WINDOWS):
-	@echo
-	CGO_ENABLED=0 GOARCH=$(ARCH) GOOS=windows GO111MODULE=on go build -ldflags='$(LDFLAGS)' -gcflags='$(GCFLAGS)' -o $(OUTDIR)/$(BINNAME_WINDOWS) ./cmd/mesheryctl/main.go
+.PHONY: build-multiple
+build-multiple: LDFLAGS += -extldflags "-static"
+build-multiple: $(GOX)
+	GO111MODULE=on CGO_ENABLED=0 $(GOX) -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' ./cmd/mesheryctl/
```

Comment:
Has `_dist` been added to .gitignore and `bin` removed?

Reply:
Thanks.